app/add/page.tsx

```diff
@@ -0,0 +1,168 @@
+"use client";
+
+import {
+	Box,
+	Button,
+	Card,
+	Container,
+	FormControl,
+	FormLabel,
+	InputLabel,
+	MenuItem,
+	Select,
+	Stack,
+	TextField,
+	Typography,
+} from "@mui/material";
+import { useTaskContext } from "../context/TaskContext";
+import { useState } from "react";
+import { useRouter } from "next/navigation";
+
+export default function Add() {
+	const { addTask } = useTaskContext();
+
+	const [type, setType] = useState("");
+	const [status, setStatus] = useState("");
+	const [errors, setErrors] = useState<{ [key: string]: string }>({});
+
+	const router = useRouter();
+
+	const handleValidation = (title: string, description: string) => {
```

Comment:
use react useForm hook for form related functionality:
https://react-hook-form.com/docs/useform